Source/Cells/Base/FORMBaseFieldCell.m

```diff
@@ -118,10 +125,25 @@ - (CGRect)separatorViewFrame {
 #pragma mark - Styling
 
 - (void)setHeadingLabelFont:(UIFont *)font {
+    NSString *styleFont = [self.styles valueForKey:@"heading_label_font"];
```

Comment:
There are a lot of strings, and strings mean someone can mistype something, could you move this strings into constants, please? 

For example:

```objc
static NSString * const FORMHeadingLabelFontKey = @"heading_label_font";
static NSString * const FORMHeadingLabelFontSizeKey = @"heading_label_font_size";
```